backstory/src/demo/webapp/gpt2.js

```diff
@@ -17,3 +16,74 @@
  */
 
 // DISPLAY STORY
+
+async function displayStory() {
+  /* eslint-disable no-unused-vars */
+  const input = document.getElementById('prompt').value;
+
+  if (input === '' || input === null) {
+    alert('You need to enter a value');
+    return;
+  }
+
+  console.log(input);
+
+  // get display
+  const display = document.getElementById('story-display');
+  display.appendChild(formatResponse('Loading...'));
+  // grab data and get its text version (it is sent as JSON)
+  const response = await fetch('/gpt2', {
+    method: 'post',
+    headers: {'Content-Type': 'application/json'},
+    body: `{text: ${input}}`,
+  });
+
+  const ok = response.ok;  // checks for server error
```

Comment:
Can this be declared below just before it's used? Keeping related code together makes it easier to follow.

Reply:
Done